Adding short tests to qmaxpool benchmarking

z-a-f opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #29274 qadaptive_avgpool2d benchmarking
* #29268 qavgpool benchmarking
* **#29257 Adding short tests to qmaxpool benchmarking**

Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)

Commit:
Adding short tests

Commit:
Update on "Adding short tests"


Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)

Commit:
Update on "Adding short tests"


Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)

Commit:
Update on "Adding short tests"


Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)

Commit:
Update on "Adding short tests"


Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)

Commit:
Update on "Adding short tests"


Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)

Commit:
Update on "Adding short tests"


Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)

mingzhe09088 commented on benchmarks/operator_benchmark/pt/qactivation_test.py:
```diff
@@ -81,8 +90,10 @@ def init(self, dims, permute_dims, inplace, dtype):
-op_bench.generate_pt_test(qactivation_configs, QReLU6Benchmark)
+op_bench.generate_pt_test(qactivation_short_configs, QReLUBenchmark)
+op_bench.generate_pt_test(qactivation_short_configs, QReLU6Benchmark)
+op_bench.generate_pt_test(qactivation_long_configs, QReLUBenchmark)
```
Configs can be concatenated. So do like this: op_bench.generate_pt_test(qactivation_short_configs + qactivation_long_configs, QReLUBenchmark)

Commit:
Update on "Adding short tests"


Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)

Commit:
Update on "Adding short tests"


Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)

Commit:
Update on "Adding short tests"


Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)

Commit:
Update on "Adding short tests"


Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)

Commit:
Update on "Adding short tests"


Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)

mingzhe09088 reviewed (approved):
LGTM!

Commit:
Rebasing on "Adding short tests to qmaxpool benchmarking"


Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)

Commit:
Update on "Adding short tests to qmaxpool benchmarking"


Differential Revision: [D18340536](https://our.internmc.facebook.com/intern/diff/D18340536)
